make/CMake_aarch64_gen_codec.cmake

```diff
@@ -35,13 +35,25 @@ if (NOT PYTHONINTERP_FOUND)
   message(FATAL_ERROR "Python interpreter not found")
 endif ()
 
+
 set(AARCH64_CODEC_GEN_SRCS
   ${PROJECT_BINARY_DIR}/opcode.h
   ${PROJECT_BINARY_DIR}/decode_gen.h
   ${PROJECT_BINARY_DIR}/encode_gen.h
   ${PROJECT_BINARY_DIR}/opcode_names.h
 )
 set_source_files_properties(${AARCH64_CODEC_GEN_SRCS} PROPERTIES GENERATED true)
+
+set(CODEC_TXT "${PROJECT_SOURCE_DIR}/core/arch/${ARCH_NAME}/codec.txt")
+
+execute_process(COMMAND "${PROJECT_SOURCE_DIR}/make/aarch64_check_codec_order.sh" ${PROJECT_SOURCE_DIR}/core/arch/aarch64 ${PROJECT_BINARY_DIR}
```

Comment:
We can't rely on bash being available on Windows -- but I guess this will never be run on Windows.  I think a `if (WIN32) message(FATAL_ERROR...` would make this clear.